### PR TITLE
fix update session and add get sessions functionality

### PR DESCRIPTION
**Pull request by acchiang**
fix broken functionality from last refactor 
now user can rename session again!

### CODE REVIEW

File: server/controllers/session-ctrl.js [hunk 1]
```diff
@@ -5,6 +5,20 @@ const generateSessionId = () => {
   return Array.from(arr, val => val.toString(16)).join("");
 };
 
+getSessions = async (req, res) => {
+  Session.find({},
+    function(err, response) {
+      if (err) {
+        console.log("Error: " + err);
```
Comment (kozr) [inline server/controllers/session-ctrl.js]: they look at the code so just get rid of it
```suggestion
```

File: server/controllers/session-ctrl.js [hunk 2]
```diff
@@ -46,8 +60,27 @@ getSessionName = async (req, res) => {
...
 }
 
+updateSessionName = async (req, res) => {
+  const body = req.body
+  const sessionId = req.params.sessionId
+  Session.findByIdAndUpdate(sessionId, {name: body.name},
+    { new: true },
+    function(err, response) {
+      if (err) {
+        console.log("Error: " + err);
```
Comment (kozr) [inline server/controllers/session-ctrl.js]:
```suggestion
```

**Review by kozr — APPROVED** [review]
Functions well

**Review by christylo — APPROVED** [review]

**Comment by acchiang** [comment]
thanks for the comments @kozr

**Pull request merged**